lib/bgsite_official_web/templates/tag/show.html.eex

```diff
@@ -0,0 +1,41 @@
+<div class="alert alert-info text-center">
+  <a href="/" class="blue-link-color"><%= @tag.title %>
+</div>
+<div class="container">
+  <div class="row">
+
+    <%= for websites <- @websites_for_tag do %>
```

Comment:
Please, keep the convention and change it to `website` as we have discussed a number of times.